### ISSUE TITLE
Export/download graph image

### ISSUE THREAD

**Issue by ghost** [pulumi/console-requests#53]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

I’d like to be able to export, or download, the graph image as an image file to my device.

### Affected area/feature

Resource graph.

**Comment by aureq** [comment]
@aviflat have you considered exporting the [stack graph](https://www.pulumi.com/docs/reference/cli/pulumi_stack_graph/) into a `.dot` format using `pulumi stack graph`? And then maybe use this to render the graph ? https://github.com/gmamaladze/d3-dot-graph

**Comment by aviflat** [comment]
@aureq yes, I tried doing that earlier today, and a basic invocation of `dot` yielded an image that was an ugly, incoherent mess.

I suppose perhaps there are a set of settings for `dot`, or perhaps an alternative dot renderer, that might yield something useful. But that’s the whole point of this request — I don’t know what those are, and I don’t have time to research this.

That said, it seems to me that this request could certainly be fulfilled by a new page in the docs, or even just some additional paragraphs on [the docs for `stack graph`](https://www.pulumi.com/docs/reference/cli/pulumi_stack_graph/) that’d provide some guidance on how to render the `.dot` file into a useful image.

(BTW it’d also be nice if the filename arg to `stack graph` was optional, in which case it’d just write to stdout. Would make it easier to construct a pipeline to render an image.)
